ci/quesma/config/ci-config.yaml

```diff
@@ -1,15 +1,33 @@
-mode: "dual-write-query-clickhouse"
 installationId: "just-Quesma-smoke-test-instance"
-port: 8080
-indexes:
-  kibana_sample_data_ecommerce:
-    enabled: true
-  kibana_sample_data_flights:
-    enabled: true
-  kibana_sample_data_logs:
-    enabled: true
-  logs-generic-default:
-    enabled: true
-    fullTextFields: [ "message" ]
-  windows_logs:   # Used for EQL e2e tests
-    enabled: true
+frontendConnectors:
+  - name: elasticsearchFakedAPI
+    type: elasticsearch-fe
+    config:
+      listenPort: 8080
+backendConnectors:
+  - name: my-minimal-elasticsearch
+    type: elasticsearch
+    config:
+      url: "http://elasticsearch:9200"
+  - name: my-clickhouse-cloud-data-source
+    type: clickhouse-os
+    config:
+      url: "clickhouse://clickhouse:9000"
+processors:
+  - name: elasticsearch-to-clickhouse-processor
+    type: quesma-v1-processor
+    config:
+      mode: "dual-write-query-clickhouse"
```

Comment:
I'm wondering whether `dual-write-query-clickhouse` effectively mean/implies that `enableElasticsearchIngest` should be true.

Reply:
WDYT about introducing `disableElasticsearchIngest` flag then?

Reply:
(yea I agree the implication is the opposite to what I'd expect)

Reply:
👍 so let it stay as is for now to maintain the default behavior and we'll clean this up as we revisit modes.